spirekey/src/app/(wallet)/layout.tsx

```diff
@@ -12,7 +12,7 @@ const BetaNotification = dynamic(
   },
 );
 
-import '../global.css';
+import './global.css';
```

Comment:
I don't see any global.css on this path?

Reply:
Is this because we miss a prefix?

Reply:
nvm, the file got moved in this pr :joy: